components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/PulsarConfiguration.java

```diff
@@ -63,6 +63,8 @@
     private long ackGroupTimeMillis = 100;
     @UriParam(label = "consumer", defaultValue = "LATEST")
     private SubscriptionInitialPosition subscriptionInitialPosition = LATEST;
+    @UriParam(label = "consumer", defaultValue = "false")
+    private boolean readCompacted = false;
```

Comment:
The defaultValue at field level is not needed, it is false by default.

Reply:
done. :)